### DISCUSSION TITLE
How to set the background color of a page?

### DISCUSSION THREAD

**Discussion by morningstarsabrina** [zauberzeug/nicegui#407] [Q&A] [answered]
How to change all the background color ?
for example with the following code:
```py

from nicegui import ui

with ui.tabs().classes('w-full').style("background-color: #98a3b0;") as tabs:
    ui.tab('Home', icon='home')
    ui.tab('About', icon='info')


with ui.tab_panels(tabs, value='Home').classes('w-full h-screen').style("background-color: #98a3b0;"):
    with ui.tab_panel('Home'):
        ui.label('This is the first tab')
    with ui.tab_panel('About'):
        ui.label('This is the second tab')

ui.run()


```
I'm getting white  between elements and on the side like this
<a href="https://ibb.co/frXrGcF"><img src="https://i.ibb.co/SV6V7Ly/Cattura.png" alt="Cattura" border="0"></a>

is it possible to set the background color for all the page ?
thanks

**Comment by rodja** [comment, marked as answer]
You can use 

```py
ui.add_head_html('<style>body {background-color: #81D4FA; }</style>')
```

to set the background color. We should simplify this. Maybe incorporating into [ui.color](https://nicegui.io/reference#color_theming) would be nice.

**Reply by morningstarsabrina** [reply]
thanks, that's what I wanted.
Yes [ui.color](https://nicegui.io/reference#color_theming)  was the first thing I tried but it didn't work but I think it make sense to put it there for simplicity 👍

**Reply by rodja** [reply]
I created #411 to track this enhancement.

**Reply by morningstarsabrina** [reply]
perfect. thanks.

**Comment by falkoschindler** [comment]
Just for the sake of completeness, you can also access the page `layout` of the current `client`, which is passed into page generator functions as an optional argument:

```py
from nicegui import Client, ui

@ui.page('/')
def page(client: Client):
    ui.label('Test 1')
    client.layout.classes('bg-blue-100')
    # or
    client.layout.style('background-color: skyblue')

ui.run()
```

**Reply by Shahin-rmz** [reply]
Thanks for answer.
we can use all CSS functions here right?
I want to have blurry image as background on my page. (Just want to see how flexible NiceGUI is)
Best

**Reply by falkoschindler** [reply]
Yes, that should work.
